website/content/docs/concepts/ha.mdx

```diff
@@ -169,12 +169,12 @@ variable, which takes precedence.
 ## Storage Support
 
 Currently there are several storage backends that support high availability
-mode, including [Consul](/docs/storage/consul),
-[ZooKeeper](/docs/storage/zookeeper) and [etcd](/docs/storage/etcd). These may
+mode, including [Consul](/docs/configuration/storage/consul),
+[ZooKeeper](/docs/configuration/storage/zookeeper) and [etcd](/docs//configuration/storage/etcd). These may
```

Comment:
There's two forward slashes in the etcd link - not sure how that affects the markup.

Reply:
Nice catch! Removed.